### DISCUSSION TITLE
配合vimium模式，在选中某个单词后，按下快捷键，弹出单词的翻译。

### DISCUSSION THREAD

**Discussion by bushnerd** [immersive-translate/immersive-translate#2218] [Ideas]
### 新功能 | New Feature

1.如题

### 为什么您想要添加这些功能 | Why you want to add these features

1.平时一般使用vimium来浏览网页，基本不使用鼠标，效率极高
2.只是偶尔碰上一些单词不认识，需要查单个单词，而全文翻译有点重
3.希望能够使用vimium的模式，visual模式下选中某个单词，按下某个快捷键，然后就直接翻译
4.试用了很多扩展，目前发现只有google translate扩展可以实现

**Comment by theowenyoung** [comment]
请问 google translate 扩展是如何配合 vimium 实现该功能的？

不过 沉浸式翻译 本身并不支持划词翻译。

**Reply by bushnerd** [reply]
我也不清楚是如何实现的，但是使用方式如下：
![1](https://github.com/user-attachments/assets/4b8a3bf5-7f14-4070-81e0-48b9e53ce7fd)

**Reply by theowenyoung** [reply]
visual模式选中之后，再按下哪个键呢？这个键是你配置的吗？还是

**Reply by bushnerd** [reply]
> visual模式选中之后，再按下哪个键呢？这个键是你配置的吗？还是

是的，是我配置的，然后这个扩展会读取当前选中的内容进行翻译。
![image](https://github.com/user-attachments/assets/a508abf7-3453-4c29-b4d7-c658d6809317)
